Refresh.GameServer/Types/Levels/Categories/DeveloperLevelsCategory.cs

```diff
@@ -0,0 +1,24 @@
+using Bunkum.Core;
+using Refresh.GameServer.Authentication;
+using Refresh.GameServer.Database;
+using Refresh.GameServer.Endpoints.Game.Levels.FilterSettings;
+using Refresh.GameServer.Services;
+using Refresh.GameServer.Types.UserData;
+
+namespace Refresh.GameServer.Types.Levels.Categories;
+
+public class DeveloperLevelsCategory : LevelCategory
+{
+    internal DeveloperLevelsCategory() : base("developer", Array.Empty<string>(), false)
```

Comment:
Uh, what? Shouldn't we be specifying an API route here?

Reply:
API route is specified, not game route

Reply:
I'm guessing it's handled as a quirk in LevelEndpoints, then?

Reply:
Theres no quirk, the fact the game endpoint array is empty means its just never considered by the LevelEndpoints code

Reply:
I'm surprised it's worked so far, then. I'll open an issue, we can investigate later